cmd/neofs-cli/modules/container/create.go

```diff
@@ -157,7 +157,7 @@ func initContainerCreateCmd() {
 	flags.StringSliceVarP(&containerAttributes, "attributes", "a", nil, "Comma separated pairs of container attributes in form of Key1=Value1,Key2=Value2")
 	flags.BoolVar(&containerAwait, "await", false, fmt.Sprintf("Block execution until container is persisted. "+
 		"Increases default execution timeout to %.0fs", awaitTimeout.Seconds())) // simple %s notation prints 1m0s https://github.com/golang/go/issues/39064
-	flags.StringVar(&containerName, "name", "", "Container name attribute")
+	flags.StringVar(&containerName, "name", "", "Container name attribute and domain name attribute, that is registered in NNS contract")
```

Comment:
...with the default zone...

It's important, for the name of `supercontainer` you get `supercontainer.container` domain in fact.